<?xml version="1.0"?>
<!--
  Gripper launch file.
  Launched 2 files: gripper connectie file, gripper service server.
-->
<launch>
  <node pkg="robotiq_2f_gripper_control" type="Robotiq2FGripperRtuNode.py"  name="robotiq2FGripper" output="log">
  </node>

  <node pkg="gripper_besturing" type="gripper_service_server.py"  name="bestruring_gripper_node"  output="screen">
  </node>
</launch>
